integration/integrate-with-image-vulnerability-scanners.adoc

```diff
@@ -32,22 +32,26 @@ This enhanced support gives you greater flexibility and choice in managing your
+** StackRox Scanner: This scanner is the default scanner in {product-title-short}. It originates from a fork of the Clair v2 open source scanner. If you have Scanner V4 enabled, you must also enable this scanner to provide scanning of RHCOS nodes and platform vulnerabilities such as {osp}, Kubernetes, and Istio. Support for that functionality in Scanner V4 is planned for a future release.
+// provide instructions for selecting this scanner to use
+* link:https://github.com/quay/clair[Clair]: As of release 4.4, you can enable Scanner V4 in {product-title-short} to provide functionality provided by ClairCore, which also powers the Clair V4 scanner. However, you can configure Clair V4 as the scanner by configuring an integration.
 * link:https://cloud.google.com/container-registry/docs/container-analysis[Google Container Analysis]
 * link:https://quay.io[Red Hat Quay]
 
 [IMPORTANT]
 ====
-{product-title-short} Scanner is the preferred image vulnerability scanner to use with {product-title-short}.
-For more information about scanning container images with {product-title-short} Scanner, see xref:../operating/examine-images-for-vulnerabilities.adoc#scanning-images_examine-images-for-vulnerabilities[Scanning images].
+The StackRox Scanner, in conjunction with Scanner V4 (optional), is the preferred image vulnerability scanner to use with {product-title-short}.
+For more information about scanning container images with the StackRox Scanner and Scanner V4, see xref:../operating/examine-images-for-vulnerabilities.adoc#scanning-images[Scanning images].
 ====
 
-If you use one of these products in your DevOps workflow, you can use the {product-title-short} portal to configure an integration with your vulnerability scanner. After the integration, the {product-title-short} portal shows the image vulnerabilities and you can triage them easily.
+If you use one of these alternative scanners in your DevOps workflow, you can use the {product-title-short} portal to configure an integration with your vulnerability scanner. After the integration, the {product-title-short} portal shows the image vulnerabilities and you can triage them easily.
+
+If multiple scanners are configured, {product-title-short} tries to use the non-StackRox/{product-title-short} and Clair scanners. If those scanners fail, {product-title-short} tries to use a configured Clair scanner. If that fails, {product-title-short} tries to use Scanner V4, if configured. If Scanner V4 is not configured or fails, {product-title-short} tries to use the StackRox Scanner.
```

Comment:
```suggestion
If multiple scanners are configured, {product-title-short} tries to use scanners that are not StackRox, {product-title-short}, or Clair. If those scanners fail, {product-title-short} tries to use a configured Clair scanner. If that fails, {product-title-short} tries to use Scanner V4, if configured. If Scanner V4 is not configured or fails, {product-title-short} tries to use the StackRox Scanner.
```

Reply:
@RTann - Can you clarify? The way I understand it is that it first tries to use any scanner that isn't StackRox, isn't Scanner V4, and isn't Clair. Trying to figure out a better way of saying this since as Jeana points out, we aren't supposed to use `/` (because it's confusing).